openapi/openapi/src/main/scala/endpoints/openapi/StatusCodes.scala

```diff
@@ -0,0 +1,31 @@
+package endpoints
+package openapi
+
+/**
+  * Interpreter for [[endpoints.algebra.StatusCodes]]
+  *
+  * @group interpreters
+  */
+trait StatusCodes extends endpoints.algebra.StatusCodes {
+
+  sealed trait StatusCode {
+    def value: Int
+    override def toString = value.toString
+  }
```

Comment:
I would just go with `type StatusCode = Int`. What is the benefit of adding a `StatusCode` class?

In addition, it should not be `sealed`: we really want users to be able to add their own status codes if they want to.

Reply:
Indeed no benefit, probably I had an idea to had something, but can't remember what. Good catch for the `sealed`